engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/emstd/CharEmStdOperator.java

```diff
@@ -0,0 +1,166 @@
+package io.deephaven.engine.table.impl.updateby.emstd;
+
+import io.deephaven.api.updateby.OperationControl;
+import io.deephaven.chunk.CharChunk;
+import io.deephaven.chunk.Chunk;
+import io.deephaven.chunk.LongChunk;
+import io.deephaven.chunk.attributes.Values;
+import io.deephaven.engine.rowset.RowSequence;
+import io.deephaven.engine.table.ColumnSource;
+import io.deephaven.engine.table.MatchPair;
+import io.deephaven.engine.table.impl.updateby.UpdateByOperator;
+import io.deephaven.engine.table.impl.util.RowRedirection;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import static io.deephaven.util.QueryConstants.*;
+
+public class CharEmStdOperator extends BasePrimitiveEmStdOperator {
+    public final ColumnSource<?> valueSource;
+    // region extra-fields
+    // endregion extra-fields
```

Comment:
needed?

Reply:
Needed by `ReplicateUpdateBy.java` for `Byte` specialization.